### ISSUE TITLE
Update docker-compose to 1.20.1 to solve build context contained directory symlinks

### ISSUE THREAD

**Issue by gaelleiadvize** [docker/for-mac#2719]
Please, check https://docs.docker.com/docker-for-mac/troubleshoot/.
Issues without logs and details cannot be debugged, and will be closed.

  - [x] I have tried with the latest version of my channel (Edge)
  - [x] I have submitted Diagnostics
  - [x] I have included the Diagnostics ID in this Issue

### Expected behavior
docker-compose build should work with directory containing symlinks
### Actual behavior
I run docker-compose build on my directory containing symlinks & it fail : 
 ```
 ERROR: Error processing tar file(exit status 1): open /public/css/html.css: no such file or directory
```
public/css is a symlink 

My docker-compose file : 
```
version: '2'

services:
  my-app:
    build: .
    volumes:
      - .:/app
  my-app-web:
    image: nginx
    container_name: my-app-web
    restart: unless-stopped
    ports:
      - 8181:80
    volumes_from:
      - my-app
    env_file: .env
```
 - This issue is fixed on last version 1.20.1 :) 
https://github.com/docker/compose/releases

 - History of the docker-compose issue : 
  https://github.com/docker/compose/issues/5809

### Information

```
Docker for Mac: version: 18.03.0-ce-rc4-mac57 (dd2831d4b7421cf559a0881cc7a5fdebeb8c2b98)
macOS: version 10.13.3 (build: 17D47)
logs: /tmp/F3DCC71B-E272-4B5B-A2A1-DF503D5161CA/20180323-084209.tar.gz
[OK]     vpnkit
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     files
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[OK]     docker-cli
[OK]     disk

```

**Comment by pgayvallet** [comment]
Hello,

Good news, the compose 1.20.1 will be shipped in the 18.03.0-GA (edge & stable), due today or at the beginning of next week.

**Comment by djs55** [comment]
The fix has been shipped on both stable and edge channels so I'll close the ticket.

Thanks for your report!
